### PR TITLE
Task #1987: dashboard UI enhancement and theme configuration

### PR DESCRIPTION
**Pull request by shreyas1434shinde**

### CODE REVIEW

File: src/components/PasswordCreate.tsx [hunk 1]
```diff
@@ -55,8 +61,72 @@ const PasswordCreate: React.FC<PasswordCreateProps> = ({
...
+              shrink: true,
+            }}
+            type={'password'}
+            value={oldPassword}
+            onChange={(e) => setOldPassword(e.target.value)}
+            error={oldPasswordError}
+            helperText={
+              oldPasswordError && t('LOGIN_PAGE.OLD_PASSWORD_INCORRECT')
+            }
+            label="Old Password"
```
Comment (itsvick) [inline src/components/PasswordCreate.tsx:119]: @shreyas1434shinde translate

File: src/components/PasswordCreate.tsx [hunk 2]
```diff
@@ -55,8 +61,72 @@ const PasswordCreate: React.FC<PasswordCreateProps> = ({
   const isFormValid =
     !passwordError && !confirmPasswordError && password && confirmPassword;
 
+  const handleFormSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (editPassword) {
+      const userIdName = localStorage.getItem('userIdName');
+      if (!userIdName) {
+        console.error('User ID not found in localStorage');
```
Comment (itsvick) [inline src/components/PasswordCreate.tsx:70]: @shreyas1434shinde give some error on toast message and logout this user

File: src/components/PasswordCreate.tsx [hunk 3]
```diff
@@ -212,8 +282,8 @@ const PasswordCreate: React.FC<PasswordCreateProps> = ({
                 width: '50%',
               },
             }}
-            onClick={() => handleResetPassword(password)}
-            disabled={!isFormValid}
+            onClick={handleFormSubmit}
```
Comment (itsvick) [inline src/components/PasswordCreate.tsx:285]: @shreyas1434shinde Add check for the old and new password and if both are same, show error that both password cannot be same

File: src/components/PasswordCreate.tsx (hunk 1 above)
Comment (shreyas1434shinde) [reply src/components/PasswordCreate.tsx]: resolved

File: src/components/PasswordCreate.tsx (hunk 3 above)
Comment (shreyas1434shinde) [reply src/components/PasswordCreate.tsx:325]: @itsvick resolved

File: src/components/PasswordCreate.tsx (hunk 2 above)
Comment (shreyas1434shinde) [reply src/components/PasswordCreate.tsx]: @itsvick resolved

File: public/locales/en/common.json [hunk 4]
```diff
@@ -167,7 +167,10 @@
     "DO_IT_LATER": "I’ll do it later",
     "SUCCESSFULLY_RESET": "Your password has been successfully reset",
     "LINK_EXPIRED": "Link Expired",
-    "WE_SEND_AN_EMAIL":"We sent an email to {{maskEmail}} with a link to reset your password"
+    "WE_SEND_AN_EMAIL":"We sent an email to {{maskEmail}} with a link to reset your password",
+    "CURRENT_PASSWORD_NOT":"Current Password Not Matched",
+    "NO_USERNAME":"N0 user name found",
```
Comment (itsvick) [inline public/locales/en/common.json:172]: @shreyas1434shinde `N0 user name found` change it to `Username not found`

File: public/locales/en/common.json [hunk 5]
```diff
@@ -167,7 +167,10 @@
...
+    "PASSWORD_SAME_AS_OLD":"Password Same as Old"
```
Comment (itsvick) [inline public/locales/en/common.json:173]: @shreyas1434shinde Password Same as Old make it `Password same as old`

Comment (itsvick) [reply public/locales/en/common.json:173]: or change this error message

File: public/locales/en/common.json (hunk 4 above)
Comment (shreyas1434shinde) [reply public/locales/en/common.json:172]: @itsvick resolved

File: public/locales/en/common.json (hunk 5 above)
Comment (shreyas1434shinde) [reply public/locales/en/common.json:173]: @itsvick resolved

**Pull request merged**